torchvision/datasets/hmdb51.py

```diff
@@ -92,24 +107,24 @@ def __init__(self, root, annotation_path, frames_per_clip, step_between_clips=1,
         self.transform = transform
 
     @property
-    def metadata(self):
+    def metadata(self) -> Dict[str, Any]:
         return self.full_video_clips.metadata
 
-    def _select_fold(self, video_list, annotations_dir, fold, train):
+    def _select_fold(self, video_list: List[str], annotations_dir: str, fold: int, train: bool) -> List[int]:
         target_tag = self.TRAIN_TAG if train else self.TEST_TAG
         split_pattern_name = "*test_split{}.txt".format(fold)
         split_pattern_path = os.path.join(annotations_dir, split_pattern_name)
         annotation_paths = glob.glob(split_pattern_path)
-        selected_files = []
+        _selected_files = []
```

Comment:
Instead of appending to a `list` and later turn it into a `set` why not start out with one?
```suggestion
        selected_files = set()
```